.circleci/config.yml

```diff
@@ -181,15 +181,22 @@ jobs:
               --template-file ~/repo/thumbs/.aws-sam/build/ThumbsFunction/template.yml \
               --region eu-west-2 \
               --s3-bucket=$THUMBS_DEPLOY_BUCKET_NAME
+              --parameter-overrides " \
+                   AppSentryDSN='$SENTRY_DSN' \
+                  AppSentryLayer='arn:aws:lambda:eu-west-2:943013980633:layer:SentryPythonServerlessSDK:137'
```

Comment:
This probably doesn't matter, but there's a bit of strange indenting here.

Also, is this going to use the same sentry DSN as the main leaflets lambda? Do we need to do anything to differentiate the two?

Reply:
Good catch, fixed up.

In terms of knowing what's what: it will be obvious from the code paths, and I think it's more useful to have them in one project than across two? I don't actually know how it will look in Sentry though, maybe it will indicate the function name. I can take a look when it's deployed

Reply:
Lets get to a stage where we have some visibility rather than none first
Then we can tweak it if we need to